src/wieldmesh.h

```diff
@@ -44,6 +44,8 @@ class WieldMeshSceneNode: public scene::ISceneNode
 			v3f wield_scale, ITextureSource *tsrc, u8 num_frames);
 	void setItem(const ItemStack &item, IGameDef *gamedef);
 
+	ItemStack getItem();
```

Comment:
that was a mistake, didn't mean to add that (I was going to get the keyframes another way, but then realised that there was no need)